src/js/components/Image.js

```diff
@@ -3,14 +3,16 @@
 import React, { Component, PropTypes } from 'react';
 import classnames from 'classnames';
 
+import Label from './Label';
+
 const CLASS_ROOT = 'image';
 
 export default class Image extends Component {
   render () {
-    let { size, full } = this.props;
+    let { caption, className, full, id, size, src } = this.props;
     let classes = classnames(
       CLASS_ROOT,
-      this.props.className,
+      className,
```

Comment:
Can we put the callers className last? It's very minor but makes inspecting the DOM easier.